tests/Issues/PartialValueDocblockUpdate/Source/PartialUpdateTestRector.php

```diff
@@ -42,6 +48,8 @@ public function refactor(Node $node)
                 'defaults',
             );
 
+            $this->docBlockUpdater->updateRefactoredNodeWithPhpDocInfo($node);
```

Comment:
/cc @shyim FYI, ref issue and PR:

- https://github.com/rectorphp/rector/issues/7020
- https://github.com/rectorphp/rector-src/pull/1862